test/python/providers/basic_provider/test_basic_simulator.py

```diff
@@ -191,7 +191,7 @@ def test_memory(self):
 
     def test_unitary(self):
         """Test unitary gate instruction"""
-        max_qubits = 4
+        max_qubits = 4  # changed back to 4
```

Comment:
Thanks for doing this. No need for explicit comment though.

```suggestion
        max_qubits = 4
```